### PR TITLE
Update db versions in docker compose template

### LINKED ISSUE #9326: Update docker image that generated by typeorm cli: `npx typeorm --init docker`
### The Problem

Docker image that generated by TypeORM CLI is old.

example:
- `npx typeorm --init docker`
- open docker-compose.yml
```yaml
version: '3'
services:

  postgres:
    image: "postgres:9.6.1"
    ports:
      - "5432:5432"
    environment:
      POSTGRES_USER: "test"
      POSTGRES_PASSWORD: "test"
      POSTGRES_DB: "test"
```

I think `image: "postgres:9.6.1"` is old. So I want to update.

### The Solution

Update `getDockerComposeTemplate()` to use `latest`.

https://github.com/typeorm/typeorm/blob/master/src/commands/InitCommand.ts#L521

### Relevant Database Driver(s)

| DB Type           | Relevant      |
|-------------------| ---           |
| `aurora-mysql`    | no            |
| `aurora-postgres` | no            |
| `better-sqlite3`  | no            |
| `cockroachdb`     | no            |
| `cordova`         | no            |
| `expo`            | no            |
| `mongodb`         | no            |
| `mysql`           | no            |
| `nativescript`    | no            |
| `oracle`          | no            |
| `postgres`        | no            |
| `react-native`    | no            |
| `sap`             | no            |
| `spanner`         | no            |
| `sqlite`          | no            |
| `sqlite-abstract` | no            |
| `sqljs`           | no            |
| `sqlserver`       | no            |

### Are you willing to resolve this issue by submitting a Pull Request?

 - ✅ Yes, I have the time, and I know how to start.
 - ✖️ Yes, I have the time, but I don't know how to start. I would need guidance.
 - ✖️ No, I don’t have the time, but I can support (using donations) development.
 - ✖️ No, I don’t have the time and I’m okay to wait for the community / maintainers to resolve this issue.

### PR DESCRIPTION
**Pull request by Jacse**
### Description of change

Fixes #9326

### Pull-Request Checklist

- [x] Code is up-to-date with the `master` branch
- [x] `npm run format` to apply prettier formatting
- [ ] `npm run test` passes with this change
- [x] This pull request links relevant issues as `Fixes #0000`
- [ ] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [x] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Jacse** [comment]
Closed in favour of #9367

File: src/commands/InitCommand.ts [hunk 1]
```diff
@@ -555,7 +555,7 @@ services:
 services:
 
   postgres:
-    image: "postgres:9.6.1"
+    image: "postgres:14.5."
```
Comment (pleerock) [inline src/commands/InitCommand.ts:558]: just want to make sure you didn't miss the number after "14.5."

**Comment by pleerock** [comment]
Thank you for contribution! 

Did you check if TypeORM works with `cockroach:v22.1.6` and `mongo:5.0.12` ? Thanks in advice.

**Comment by pleerock** [comment]
Which one should I merge? This one or #9367 ?

**Comment by Jacse** [comment]
@pleerock the other one, this one didn't adhere to commit conventions

**Pull request closed without merging**